tools/jira_cmd.py

```diff
@@ -521,6 +535,9 @@ def handle_sprint_update(args, jiraTool, issues):
 
 def main(args):
     j = jira_netrc_login(args.netrc)
+    max_results = args.max_results
+    if args.max_results == MAX_RESULTS and args.linked_issues:
```

Comment:
Maybe let's add a warning log to indicate of this limit?

Reply:
ok